[2025-05-13T21:09:01.759+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-13T21:09:01.780+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T21:08:43.070074+00:00 [queued]>
[2025-05-13T21:09:01.791+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T21:08:43.070074+00:00 [queued]>
[2025-05-13T21:09:01.792+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-05-13T21:09:01.813+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): process_alny> on 2025-05-13 21:08:43.070074+00:00
[2025-05-13T21:09:01.863+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4143) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-13T21:09:01.866+0000] {standard_task_runner.py:72} INFO - Started process 4144 to run task
[2025-05-13T21:09:01.865+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sec_filing_scheduler', 'process_alny', 'manual__2025-05-13T21:08:43.070074+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/orchestrate_clinical_pipeline.py', '--cfg-path', '/tmp/tmpx_i4ac2j']
[2025-05-13T21:09:01.867+0000] {standard_task_runner.py:105} INFO - Job 108: Subtask process_alny
[2025-05-13T21:09:01.927+0000] {task_command.py:467} INFO - Running <TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T21:08:43.070074+00:00 [running]> on host c1a7243cf3b7
[2025-05-13T21:09:02.046+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sec_filing_scheduler' AIRFLOW_CTX_TASK_ID='process_alny' AIRFLOW_CTX_EXECUTION_DATE='2025-05-13T21:08:43.070074+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-13T21:08:43.070074+00:00'
[2025-05-13T21:09:02.048+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-13T21:09:02.159+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-13T21:09:02.166+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-05-13T21:09:03.387+0000] {logging_mixin.py:190} INFO - Found 142 8-K filings
[2025-05-13T21:09:03.387+0000] {logging_mixin.py:190} INFO - Found 13 10-K filings
[2025-05-13T21:09:03.388+0000] {logging_mixin.py:190} INFO - Found 37 10-Q filings
[2025-05-13T21:09:03.389+0000] {logging_mixin.py:190} INFO - Found 1 new filings.
[2025-05-13T21:09:03.389+0000] {logging_mixin.py:190} INFO - Processing 1 filings...
[2025-05-13T21:09:03.390+0000] {logging_mixin.py:190} INFO - Processing filing: 0001178670-25-000062
[2025-05-13T21:09:03.523+0000] {logging_mixin.py:190} INFO - No sections found in 8-K filing 0001178670-25-000062
[2025-05-13T21:09:03.524+0000] {logging_mixin.py:190} INFO - Finished processing: 0001178670-25-000062
[2025-05-13T21:09:03.525+0000] {logging_mixin.py:190} INFO - Finished 1/1: 0001178670-25-000062
[2025-05-13T21:09:04.513+0000] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-05-13T21:09:04.605+0000] {logging_mixin.py:190} INFO - Schema validation passed.
[2025-05-13T21:09:06.605+0000] {logging_mixin.py:190} INFO - Updated processed accessions in GCS.
[2025-05-13T21:09:06.608+0000] {logging_mixin.py:190} INFO - Deleted output_dir: sec_data/ALNY
[2025-05-13T21:09:06.608+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-13T21:09:06.622+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-13T21:09:06.623+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sec_filing_scheduler, task_id=process_alny, run_id=manual__2025-05-13T21:08:43.070074+00:00, execution_date=20250513T210843, start_date=20250513T210901, end_date=20250513T210906
[2025-05-13T21:09:06.677+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-13T21:09:06.710+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-13T21:09:06.715+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
